packages/compass/src/main/application.ts

```diff
@@ -131,7 +131,11 @@ class CompassApplication {
 
     await this.setupCORSBypass();
     void this.setupCompassAuthService();
-    this.setupAutoUpdate();
+    // TODO(COMPASS-7618): For now don't setup auto-update in CI because the
+    // toasts will obscure other things which we don't expect yet.
+    if (!process.env.CI) {
```

Comment:
Thinking some more this might be true for unit tests too. Might have to narrow it more.

Reply:
We usually check for the APP_ENV in this case I think, if that's if has TODO, that's probably good enough for here